src/network/handler/InventoryTransactionHandler.ts

```diff
@@ -189,7 +191,7 @@ export default class InventoryTransactionHandler
 
                             soundPk.extraData = server
                                 .getBlockManager()
-                                .getRuntimeWithMeta(blockId, blockMeta); // In this case refers to block runtime Id
```

Comment:
This should be reverted.

Reply:
cannot be reverted as we don't have meta anymore in our sub chunks... this should be fully rewritten and would make more sense to rewrite with the new inv system